docs/notebooks/1_specz_catalogs.ipynb

```diff
@@ -0,0 +1,258 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img align=\"left\" src = \"images/linea.png\" width=140 style=\"padding: 20px\"> \n",
+    "<img align=\"left\" src = \"images/rubin.png\" width=180 style=\"padding: 30px\"> \n",
+    "\n",
+    "# Photo-z Server\n",
+    "## Tutorial Notebook 1 - Spec-z Catalogs\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Contact author: [Julia Gschwend](mailto:julia@linea.org.br)\n",
+    "\n",
+    "Last verified run: **2024-Jul-04**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Introduction \n",
+    "\n",
+    "Welcome to the PZ Server tutorials. If you are reading this notebooks for the first time, we recommend to not skip the introduction notebook: `0_introduction.ipynb` also available in this same repository. \n"
```

Comment:
It should be "this notebook" or "these notebooks". In  "we recommend to not skip the introduction notebook", i think the correct sentence is "we recommend not to skip the introduction notebook".